resources/views/pages/Public/App.svelte

```diff
@@ -7,4 +7,6 @@
     import { page } from "@inertiajs/inertia-svelte";
 </script>
 
-<div class="grid h-screen-no-nav place-items-center border border-gray-200 border-gray-700 bg-gray-800 p-3 shadow-md" />
+<div class="grid h-screen-no-nav place-items-center border border-gray-200 border-gray-700 bg-gray-800 p-3 shadow-md">
+  <h1>Hello World</h1>
```

Comment:
Change left over from testing

Reply:
Reverted change to previous line on this